Automattic/jetpack
# AI Assistant: Disable first Breve hover on mobile

Fixes #38525

## Proposed changes:

* On first hover, on mobile only, disables the popover so that it is not displayed above the sidebar

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor of a site with Breve enabled on a smartphone
* Add some text that has a highlight
* Tap on the highlighted text
* Check that the sidebar is displayed, but the popover is not
* Close the sidebar
* Tap the highlight again
* Check that the popover is displayed, but not the sidebar
* Save and reload
* Tap the highlight again
* Check that the popover is displayer, but not the sidebar
* Check that there is no regression on desktop
